server/backends/invocationdb/invocationdb.go

```diff
@@ -61,6 +63,28 @@ func (d *InvocationDB) InsertOrUpdateInvocation(ctx context.Context, ti *tables.
 	})
 }
 
+func (d *InvocationDB) UpdateInvocationACL(ctx context.Context, invocationID string, acl *aclpb.ACL) error {
+	p, err := perms.ToPerms(acl)
+	if err != nil {
+		return err
+	}
+	return d.h.Transaction(func(tx *gorm.DB) error {
+		var in tables.Invocation
+		if err := tx.Select("perms", "user_id", "group_id").Where("invocation_id = ?", invocationID).First(&in).Error; err != nil {
```

Comment:
would prefer to use raw queries rather than gormy things -- shouldn't be much more code

Reply:
Done